init.lua

```diff
@@ -56,8 +56,10 @@ skybox.set = function(player, number)
 				textures = textures,
 				clouds = true
 			})
-			player:set_sun({visible = false, sunrise_visible = false})
-			player:set_moon({visible = false})
+			-- making the sun invisible deactivates the dynamic shadows
+			-- making the sun visible with an invisible texture keeps shadows intact
+			player:set_sun({visible = true, sunrise_visible = false, texture = "skybox_invisible.png"})
+			player:set_moon({visible = true, texture = "skybox_invisible.png"})
```

Comment:
`empty.png` is provided my Minetest. Might that be of interest for you?

Reply:
I tried using empty.png as texture, but it wasn't found:

`ERROR[Main]: generateImage(): Could not load image "empty.png" while building texture; Creating a dummy image`

Reply:
My bad. It's called `blank.png`.
https://github.com/minetest/minetest/blob/master/textures/base/pack/blank.png

Reply:
using `blank.png` now, see latest commit